### ISSUE TITLE
Need a 1.10.2 Branch

### ISSUE THREAD

**Issue by SwatyB** [GameModsBR/VoiceChat#2]
I try to change the build.gradle file, the compilation is Successfull but I can't launch the mods with my games. 
```
My gradle configuration : 
minecraft {
    version = "1.10.2-12.18.3.2185"
    runDir = "run"
    mappings = 'stable_29'

    replace "@VERSION@", project.version
    replace "@MC_VERSION@", project.minecraft.version
}
```
I change the src/main/java/net/gliby/voicechat/VoiceChat.java : 

```
@Mod(
        modid = VoiceChat.MOD_ID,
        name = "Gliby\'s Voice Chat Mod",
        version = "@VERSION@",
        acceptedMinecraftVersions = "[1.10.2]"
)
```

I try to change the mcmod.info too. 

The compilation to jar with the command gradlew.bat jar works sucessfull.

```
---- Minecraft Crash Report ----
// But it works on my machine.

Time: 01/01/17 21:57
Description: There was a severe problem during mod loading that has caused the game to fail

net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from Â§aGliby'sÂ§f Voice Chat Mod (gvc)
Caused by: java.lang.NoSuchMethodError: net.minecraft.client.Minecraft.getMinecraft()Lnet/minecraft/client/Minecraft;
	at net.gliby.voicechat.client.VoiceChatClient.preInitClient(VoiceChatClient.java:151)
	at net.gliby.voicechat.VoiceChat.preInit(VoiceChat.java:91)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at net.minecraftforge.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:600)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:243)
	at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:221)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:145)
	at net.minecraftforge.fml.common.Loader.preinitializeMods(Loader.java:614)
	at net.minecraftforge.fml.client.FMLClientHandler.beginMinecraftLoading(FMLClientHandler.java:257)
	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:439)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:351)
	at net.minecraft.client.main.Main.main(SourceFile:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- System Details --
Details:
	Minecraft Version: 1.10.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_25, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 158135936 bytes (150 MB) / 360726528 bytes (344 MB) up to 1060372480 bytes (1011 MB)
	JVM Flags: 6 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx1G -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	FML: MCP 9.32 Powered by Forge 12.18.3.2185 4 mods loaded, 4 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCH	mcp{9.19} [Minecraft Coder Pack] (minecraft.jar) 
	UCH	FML{8.0.99.99} [Forge Mod Loader] (forge-1.10.2-12.18.3.2185.jar) 
	UCH	Forge{12.18.3.2185} [Minecraft Forge] (forge-1.10.2-12.18.3.2185.jar) 
	UCE	gvc{0.6.1} [Â§aGliby'sÂ§f Voice Chat Mod] (VoiceChat-1.10.2-0.6.1.jar) 
	Loaded coremods (and transformers): 
	GL info: ' Vendor: 'NVIDIA Corporation' Version: '3.3.0' Renderer: 'NVS 2100M/PCIe/SSE2'
```
I try with the master branch and the 1.9
 
I have no idea how does it happen. 

My computer configuration :  
Windows 10 - Java JDK 8u111 - And an official Mojang account

Regards

**Comment by Gliby** [comment]
https://github.com/GameModsBR/VoiceChat/tree/1.11.2

**Comment by Gliby** [comment]
https://github.com/GameModsBR/VoiceChat/tree/1.10.2

**Comment by joserobjr** [comment]
Did you build yourself? The stack trace shows unobfuscated names, `net.minecraft.client.Minecraft.getMinecraft()` should be named `net.minecraft.client.Minecraft.func_71410_x()`, I think you've installed an unobfuscated version to an obfuscated environment.

The 1.9 version works fine on 1.10.2, I'm using it here. I'm using [this build](http://ci.gamemods.com.br/job/VoiceChat/job/1.9/3/) as I write.

**Comment by Gliby** [comment]
Huh, I guessed the 1.9 version would have some incompatibilities with 1.10, feelsbadman. In this case new branches should be removed, so new features could be implemented ranging from 1.9 to 1.11+.
